src/GitHub.VisualStudio/Menus/MenuBase.cs

```diff
@@ -46,6 +46,24 @@ protected MenuBase(IGitHubServiceProvider serviceProvider)
             apiFactory = new Lazy<ISimpleApiClientFactory>(() => ServiceProvider.TryGetService<ISimpleApiClientFactory>());
         }
 
+        protected ILocalRepositoryModel GetRepositoryByPath(string path)
+        {
+            try
+            {
+                if (!string.IsNullOrEmpty(path))
+                {
+                    var repo = ServiceProvider.TryGetService<IGitService>().GetRepository(path);
+                    return new LocalRepositoryModel(repo.Info.WorkingDirectory.TrimEnd('\\'));
```

Comment:
CI is failing due to the assembly signing in `GitHub.InlineReview.csproj` - we should merge that PR first to get CI working.